imageio/imageio-psd/src/test/java/com/twelvemonkeys/imageio/plugins/psd/PSDImageReaderTest.java

```diff
@@ -532,4 +534,41 @@ public void testReadUnicodeLayerName() throws IOException {
             assertEquals("If_The_Layer_Name_Is_Really_Long_Oh_No_What_Do_I_Do", ((IIOMetadataNode) layerInfo.item(0)).getAttribute("name"));
         }
     }
+
+    @Test
+    public void testGroupLayerRead() throws IOException {
+        PSDImageReader imageReader = createReader();
+
+        try (ImageInputStream stream = ImageIO.createImageInputStream(getClassLoaderResource("/psd/group_layer.psd"))) {
```

Comment:
Can you please provide a smaller test file? Both in image dimensions and file size. For the test, a 16x16 image would suffice and save about 7 MB of data.